source/upgrade.rst

```diff
@@ -65,7 +65,8 @@ Docker
        docker pull qiime2/core:2017.6
 
 Amazon AWS
-    In a shell on the remote AWS machine, run the following command:
+    If you did not create your AWS instance using one of the :doc:`official QIIME 2 AMIs <install/virtual/aws>`, you should refer  to the Linux (Native Installation) upgrading instructions.
```

Comment:
Extra space between "refer" and "to." Also, can you indicate at the end of this sentence that the instructions referenced are available above?